api/src/middleware/__tests__/competenciesRouter.ts

```diff
@@ -104,4 +109,80 @@ describe('competenciesRouter', () => {
         .expect(500, done);
     });
   });
+
+  describe('PUT /:id', () => {
+    it('should update a competency of user who authored it', done => {
+      const principalId = 2;
+      const competencyId = 3;
+      const label = 'label';
+      const description = 'description';
+      const competency = {
+        id: competencyId,
+      };
+      mockPrincipalId(principalId);
+      mockAuthorizeCompetencyUpdate.mockResolvedValue(true);
+      mockUpdateCompetency.mockResolvedValue();
```

Comment:
The updateCompetency function doesn't return any values.  Do we need to mock a resolved value in this test?

Reply:
Based on the discussion on slack i will remove this mock function call